viewer/packages/rendering/src/rendering/EffectRenderManager.ts

```diff
@@ -610,8 +610,14 @@ export class EffectRenderManager {
   }
 
   private renderCustomObjects(scene: THREE.Scene, camera: THREE.PerspectiveCamera) {
-    this._renderer.setRenderTarget(this._customObjectRenderTarget);
-    this.renderStep('customobjects', scene, camera);
+    const originalAutoUpdate = scene.autoUpdate;
+    try {
+      scene.autoUpdate = true;
+      this._renderer.setRenderTarget(this._customObjectRenderTarget);
```

Comment:
Is there a scenario where setting custom render target or `renderStep` throw an exception?

Reply:
Yes, renderStep can throw @pramodcog